mission/config/subconfigs/vehicle_respawn_info.hpp

```diff
@@ -1,491 +1,1606 @@
-class respawn_short
-{
-	respawnType = "RESPAWN";
-	time = 10;
-};
+#include "..\ui\icons.inc"
```

Comment:
this file is missing in the branch, needs committing otherwise server will never boot up (@Huckleberry87 has said he's on it)

Reply:
resolved